docs/migrations/v5-to-v6.md

```diff
@@ -0,0 +1,146 @@
+# Migrating from ibc-go v5 to v6
+
+This document is intended to highlight significant changes which may require more information than presented in the CHANGELOG.
+Any changes that must be done by a user of ibc-go should be documented here.
+
+There are four sections based on the four potential user groups of this document:
+- Chains
+- IBC Apps
+- Relayers
+- IBC Light Clients
+
+**Note:** ibc-go supports golang semantic versioning and therefore all imports must be updated to bump the version number on major releases.
+
+## Chains
+
+- No relevant changes were made in this release.
+
+## IBC Apps
+
+### ICS27 - Interchain Accounts
+
+The `ibc-go/v6` release introduces a new set of migrations for `27-interchain-accounts`. Ownership of ICS27 channel capabilities is tranferred from authentication modules and will now reside with the ICS27 `controller` submodule moving forward. 
+
+For chains which implement custom authentication modules using the ICS27 `controller` submodule this requires a migration function to be included in the application upgrade handler. A subsequent migration handler is run automatically, asserting the ownership of ICS27 channel capabilities has been transferred successfully.
+
+This migration facilitates the addition of the ICS27 `controller` submodule `MsgServer` which provides a standardised approach to integrating existing forms of authentication such as `x/gov` and `x/group` provided by the Cosmos SDK. 
+
+For more information please refer to [ADR 009](https://github.com/cosmos/ibc-go/blob/main/docs/architecture/adr-009-v6-ics27-msgserver.md).
+
+#### Upgrade proposal
```

Comment:
Should the upgrade proposal be under the `Chains` section of this doc?

Reply:
I think the `Chains` section seems like pretty good place for this.